news_update.php

```diff
@@ -69,17 +69,15 @@
 
 echo '<div class="space-20"></div>';
 echo '<div class="alert alert-success">';
-echo lang_get( 'operation_successful' ) ?>
-
-echo '<br />';
+echo lang_get( 'operation_successful' );
+echo '<br /></div>';
 
 print_button( 'news_edit_page.php?news_id=' . $f_news_id . '&action=edit', lang_get( 'edit_link' ) );
+echo '&nbsp;&nbsp;&nbsp;&nbsp;';
```

Comment:
To start with, on what terms do we use Ace-Responsive? I could not find it's license in Mantis files.
How should we update it? It sure does and will have bugs.